lib/Doctrine/CouchDB/HTTP/MultipartParserAndSender.php

```diff
@@ -82,72 +80,40 @@ public function __construct(
      * @throws HTTPException
      * @throws \Exception
      */
-    public function request(
-        $sourceMethod,
-        $sourcePath,
-        $targetPath,
-        $sourceData = null,
-        array $sourceHeaders = array()
-    ) {
-        $this->sourceConnection = $this->sourceClient->getConnection(
-            $sourceMethod,
-            $sourcePath,
-            $sourceData,
-            $sourceHeaders
-        );
+    public function transferDocuments($sourceMethod, $sourcePath, $targetPath, $sourceData = null, array $sourceHeaders = array())
```

Comment:
As discussed, lets get a request method added back that just aliases this method.